### PR TITLE
Add module_permissions module to composer.json

### PR DESCRIPTION
**Pull request by guanxiaohua2k6**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -109,6 +109,7 @@
     "drupal/openid_connect": "3.x-dev",
     "drupal/queue_ui": "^3",
     "drupal/roleassign": "^2",
+    "drupal/module_permissions": "^3",
```
Comment (naoi) [inline composer.json]: @guanxiaohua2k6 Please this item in the alphabetical order.  Thanks

**Review by naoi — CHANGES_REQUESTED** [review]
@guanxiaohua2k6 Thank you for the PR.  I posted my comment.  Thanks!

**Review by naoi — CHANGES_REQUESTED** [review]
@guanxiaohua2k6 Thank you for the PR.  I posted my comment.  Thanks!

**Review by naoi — CHANGES_REQUESTED** [review]
@guanxiaohua2k6 Thank you for the PR.  I posted my comment.  Thanks!

**Comment by guanxiaohua2k6** [comment]
@naoi I change the order. Please check it again. Thanks.

**Review by naoi — APPROVED** [review]
@guanxiaohua2k6 Thank you for the update.  It looks good to me.  I'll merge this PR.  Thanks

**Pull request merged**